chart/templates/base.yaml

```diff
@@ -285,6 +285,15 @@ spec:
           required:
           - routes
           properties:
+            retryBudget:
+              type: object
+              properties:
+                retryRatio:
+                  type: number
+                minRetriesPerSecond:
+                  type: integer
+                ttl:
+                  type: string
```

Comment:
I'm not sure if `retryRatio` and `minRetriesPerSecond` are mandatory, but it seems `ttl` is, otherwise this would fail:
https://github.com/linkerd/linkerd2/blob/master/pkg/profiles/profiles.go#L61-L77
so we'd need something like:
```
         properties:
            retryBudget:
              type: object
              required:
              - ttl
              properties:
                retryRatio:
                  type: number
                minRetriesPerSecond:
                  type: integer
                ttl:
                  type: string
```

Reply:
... or better yet refactor `profiles.ToServiceProfile()` to apply a default when this field is missing.

Reply:
For the numeric fields (retryRatio and minRetriesPerSecond) it is impossible to distinguish between an explicit 0 value and the field not being present.  Therefore, for those fields, it is impossible to apply a default when the field is missing.  For consistency, I think the ttl field should work the same way.  

I think what that means is that all three fields should be required.